public/locales/en-US.json

```diff
@@ -163,6 +176,8 @@
     "legendTooltip_Storage-gibibyte-months": "Data storage usage",
     "legendTooltip_threshold_Cores_OpenShift Container Platform": "Maximum capacity, as CPU cores, based on total {{product}} Annual subscriptions in this account.",
     "legendTooltip_threshold_Cores_rosa": "Maximum capacity, as vCPU hours, based on total ROSA Hosted Control Planes pre-paid subscriptions in this account.",
+    "legendTooltip_threshold_Managed-nodes": "Maximum capacity, based on total Ansible managed nodes pre-paid subscriptions in this account.",
```

Comment:
Maximum capacity, as managed nodes, based on the amount of pre-paid nodes purchased in all subscriptions in this account.